src/PhpSpreadsheet/Writer/Xlsx/Chart.php

```diff
@@ -1144,6 +1144,13 @@ private function writePlotGroup($plotGroup, $groupType, $objWriter, &$catIsMulti
                 if ($groupType == DataSeries::TYPE_STOCKCHART) {
                     $objWriter->startElement('a:noFill');
                     $objWriter->endElement();
+                } elseif ($plotLabel && $plotLabel->getFillColor() !== null) {
+                    $fillColor = $plotLabel->getFillColor();
+                    $objWriter->startElement('a:solidFill');
+                    $objWriter->startElement('a:srgbClr');
+                    $objWriter->writeAttribute('val', $fillColor);
```

Comment:
As PHPStan correctly reports, we need to handle the case where `getFillColor()` is an array. Also this should be covered by tests.